src/modules/awake/Awake/Awake.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\..\..\Version.props" />
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>net6.0-windows</TargetFramework>
+    <TargetFramework>net6.0-windows10.0.17763.0</TargetFramework>
```

Comment:
The current minimum we're supporting is 18362.

Reply:
@jaimecbernardo I can fix that!